Source/Modules/javascript.cxx

```diff
@@ -3024,3 +3036,69 @@ String *TypeScriptTypes::getTypescriptType(Node *n)
   return typescriptType;
 }
 
+/**
+ * In some cases the complete name of the type does not match to the one
+ * existing in the symbol table due to lacking of "(" after "<" in
```

Comment:
Sorry, I'm pretty sure this is wrong. SWIG has its own type names which use `(` and ')` and there is an existing set of functions for transforming them to/from C++ type names and you should use them instead of trying to reinvent them here. I'm not sure what exactly do you need, but please have a look at `Source/Swig/typeobj.c`.

Reply:
After doing severall atempts I did not found a way to do it. Last commit works with tests and opscore, even if I hate the way I did it (with Replace and appending int to unsigned). Need to find a better way. Do you think it can be seen as an improvement or should I invest some more time in it?